app/lib/trade_tariff_backend.rb

```diff
@@ -246,5 +247,13 @@ def opensearch_host
     def opensearch_debug
       ENV.fetch('OPENSEARCH_DEBUG', 'false') == 'true'
     end
+
+    def revision
+      @revision ||= begin
+        File.read(REVISION_FILE).chomp if File.file?(REVISION_FILE)
+      rescue Errno::EACCES
```

Comment:
Is this a no permission error from the operating system? Do we want to also check if the error is missing file, etc? `ENOENT`

Reply:
Or just rescue the superclass of these?

```ruby
 Errno::EACCES.superclass
=> SystemCallError
```

Reply:
This is the same code as we were previously using in the healthcheck endpoint and it seemed to work fine there, there are specs for verifying it works with both a missing file or an unreadable (permissions) file?

I'm inclined to go with what we've already been using rather then introducing change to this part.